content/en/internal/_index.md

```diff
@@ -9,6 +9,11 @@ menu:
     pre: <i class='fa fa-cogs'></i>
 ---
 
-This section contains documentation about the EGI Internal Services, that are
-being operated centrally on behalf of EGI and supporting the coordination of
-the Federation.
+This section contains documentation about the
+[internal EGI services](https://www.egi.eu/internal-services/).
+These services are being operated centrally on behalf of EGI, and are
+supporting the coordination of the EGI Federation.
+
+{{% alert title="Note" color="info" %}} See the [User Guides](../users)
+section for documentation about public EGI services.
+{{% /alert %}}
```

Comment:
Maybe it would be good to also link to the providers part of the documentation?

Reply:
linked